src/commands/build_and_push_image.yml

```diff
@@ -201,6 +201,12 @@ parameters:
       The alias, key ID, or full ARN of the KMS key can be specified.
     default: ""
 
+  use_credentials_helper:
+    type: boolean
+    default: true
+    description: >
+      When true the authentication to docker registry will be done using the ecr-credential-helper. This avoids having
+      the password saved in text plain. Otherwise it will use the classic docker login command which is more compatible.
```

Comment:
I'd reword this to `plain text` rather than `text plain`

Reply:
Updated